apps/www/components/Climatelab/config.js

```diff
@@ -2,6 +2,7 @@ export const climateColors = {
   light: {
     default: '#5A47E1',
     text: '#FFFFFF',
+    textSoft: '#FFFFFF',
```

Comment:
are you sure it makes sense to set `textSoft` to be the same as `text`? Maybe we can just keep using the `text` color, then?

Reply:
outside of the climate lab landing page I use the real textSoft colour